infra/application.properties

```diff
@@ -7,7 +7,7 @@ debezium.source.database.user=%database_user%
 debezium.source.database.password=%database_pass%
 debezium.source.database.dbname=%database_dbname%
 debezium.source.database.server.name=api
-debezium.source.table.include.list=public.comment,public.comment_upvote,public.source_request,public.upvote,public.post,public.user,public.post_report,public.alerts,public.source_feed,public.settings
+debezium.source.table.include.list=public.comment,public.comment_upvote,public.comment_mention,public.source_request,public.upvote,public.post,public.user,public.post_report,public.alerts,public.source_feed,public.settings
```

Comment:
I think we have another file for the local debezium setup. You may want to update it there as well